### DISCUSSION TITLE
[Post-MVP]: Moving Runs from arbitrary Leaderboard to another

### DISCUSSION THREAD

**Discussion by ero-qt** [leaderboardsgg/leaderboard-backend#163] [Ideas]
Say a leaderboard contains an `Any% w/ Some Item` category that contains an arbitrary number of runs.  
Assume now that the game for this leaderboard receives new items and the moderators decide to make the used `Item` a variable instead, deciding that the items don't obsolete each other.  
The runs in that category must now be able to be moved from `Any% w/ Some Item` to `Any%` with `Some Item` as the category's new `Item` variable.

**Comment by TheTedder** [comment]
Yeah we should have a whole "admin view" that allows for batch ops like this.
